### ISSUE TITLE
Deploy Raiden Smart contract

### ISSUE THREAD

**Issue by MichaelKuo0905** [raiden-network/raiden#2956]
Dear all,
If I successfully deploy the Raiden contracts on the private chain(created by ganache), could I still use HTTP request to interact with Raiden API like what the tutorial says?  

Michael

**Comment by CosminNechifor** [comment]
Hi @MichaelKuo0905 , 

This doesn't classify as an issue, from now on when you have a question please ask them on the gitter chat [here](https://gitter.im/raiden-network/raiden) and we will reach you and answer the question. But since you asked so kindly I would give you an answer: If the contracts are deployed correctly on your private chain then when you start Raiden you would have to specify the ``endpoint-registry-contract-address``, ``tokennetwork-registry-contract-address`` and ``secret-registry-contract-address``, also the RPC endpoing of your private chain node that you are using.  There is no relation between the HTTP requests and the contracts (you don't interact directly using HTTP with the Blockchain). 

In short, if you deployed your contracts correctly and added the proper parameters when starting raiden, you will be able to interact using **HTTP with the Raiden node**. 

Please use the gitter channel from now on.  

Closing this because it's not an issue, or a feature request.
